server.go

```diff
@@ -336,12 +336,6 @@ func WithLogHandler(handler slog.Handler) func(*Server) {
 	}
 }
 
-// WithRequestContentType sets the accepted content types for the server.
-// By default, the accepted content types is */*.
-func WithRequestContentType(consumes ...string) func(*Server) {
-	return func(s *Server) { s.acceptedContentTypes = consumes }
-}
-
```

Comment:
Why is it deleted?

Reply:
My bad, moved to an engine option!

Reply:
yeah, I thought about keeping it, but since it's very broken I didn't really see the point of deprecating it.